optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/valuerange/buildin/temporal/TemporalValueRange.java

```diff
@@ -0,0 +1,144 @@
+package org.optaplanner.core.impl.domain.valuerange.buildin.temporal;
+
+import org.optaplanner.core.impl.domain.valuerange.AbstractCountableValueRange;
+import org.optaplanner.core.impl.domain.valuerange.util.ValueRangeIterator;
+import org.optaplanner.core.impl.solver.random.RandomUtils;
+
+import java.time.temporal.Temporal;
+import java.time.temporal.TemporalUnit;
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+import java.util.Random;
+
+/**
+ * Created by kevin on 08.04.2016.
+ */
```

Comment:
Removing this, as we don't keep author tags in the files (it gets out of date, instead we use Git history which is 100% accurate + we mention contributors in release notes + in team page).